pkg/controller/daemonset/daemonset_controller.go

```diff
@@ -84,7 +84,7 @@ var (
 
 	// A TTLCache of pod creates/deletes each ds expects to see
 	expectations       = kubecontroller.NewControllerExpectations()
-	updateExpectations = kruiseExpectations.NewUpdateExpectations(revisionadapter.NewDefaultImpl())
+	updateExpectations = kruiseExpectations.NewUpdateExpectations(&revisionAdapterImpl{})
```

Comment:
`daemonset` does not need to implement a new `revisionAdapterImpl`, for it has no extra feature like short hash. Just the default impl is enough.

Reply:
maybe should check it, i remember  use default revision adapter  will cause some problems.

Reply:
Can you describe the problems you meet?